src/domains/idpay-app/12_alert.tf

```diff
@@ -178,6 +178,69 @@ resource "azapi_resource" "alert_CosmosServerSideRetry" {
   })
 }
 
+# alert rule set with azapi
+resource "azapi_resource" "alert_CreateTransaction" {
```

Comment:
@larissaaiala The system has migrated to azurerm version 3. Could you check if you can use a better, more specific resource to create the alarm?